app/lib/methods/actions.ts

```
@@ -169,4 +174,4 @@ export async function triggerAction({
 	} finally {
 		invalidateTriggerId(triggerId);
 	}
-}
+}
```

ESLint

[error] Insert `⏎`

(prettier/prettier)